src/pipecat/transports/services/daily.py

```diff
@@ -1032,11 +1137,19 @@ async def cleanup(self):
     async def send_message(self, frame: TransportMessageFrame | TransportMessageUrgentFrame):
         await self._client.send_message(frame)
 
-    async def write_raw_audio_frames(self, frames: bytes):
-        await self._client.write_raw_audio_frames(frames)
+    async def register_video_destination(self, destination: str):
+        logger.warning(f"{self} registering video destinations is not supported yet")
```

Comment:
👍

Reply:
I haven't added custom video tracks in daily-python yet.